bson/src/test/unit/org/bson/codecs/pojo/TypeDataTest.java

```diff
@@ -79,37 +77,4 @@ public void testRecursiveTypeData() {
 
         typeData.toString();
     }
-
-    @Test(expected = IllegalStateException.class)
-    public void testListNoParamsValidation() {
```

Comment:
TypeData no longer validates the types, allowing users to provide custom ways to support the types, or the default  `PropertyCodecProviders` will validate them.